src/fhirtypes/ElementDefinition.ts

```diff
@@ -823,6 +826,9 @@ export class ElementDefinition {
     if (isEmpty(validTypes)) {
       throw new CodedTypeNotFoundError(this.type ? this.type.map(t => t.code) : []);
     }
+    if (units && !validTypes.find(t => t.code === 'Quantity')) {
+      throw new InvalidUnitsError(this.id);
```

Comment:
I got caught up in the peer pressure. All the other errors are throwing. Why do I only get to _log_, it's not fair!!

Reply:
Just kidding though, makes sense. The other errors are more explicitly incorrect, where in this case we can still set a potentially valid binding/fixed value. I will log instead of throwing.

Reply:
I haven't looked at the others, but I _think_ the main difference is that their errors cause an impasse -- there is no way to apply the rule without creating invalid SD.  When this error happens, however, the rule can often still be safely applied.